pkg/cluster/connection_pooler.go

```diff
@@ -402,6 +415,12 @@ func (c *Cluster) generateConnectionPoolerPodTemplate(role PostgresRole) (
 		},
 	}
 
+	if spec.TLS != nil && spec.TLS.SecretName != "" && spec.SpiloFSGroup != nil {
+		podTemplate.Spec.SecurityContext = &v1.PodSecurityContext{
+			FSGroup: spec.SpiloFSGroup,
```

Comment:
Why should it get the spilo FS group btw? Can it also get it's own? Would an extra config option make sense?

Reply:
you mean additional fsGroup in CRD pooler block?

Reply:
There could be extra settings in the [pooler struct](https://github.com/zalando/postgres-operator/blob/master/pkg/apis/acid.zalan.do/v1/postgresql_type.go#L237), yes. But, on the other hand we're already re-using TLS settings from postgrsql.spec - so I'm fine by copying SpiloFSGroup, too, as it would be convenient to users of this feature. No extra config necessary in the manifest.

Was just wondering if somebody might want this FSGroup setting to be different from spilo...